simtools/util/tests.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+
+CONFIG_FILE_MSG = "Configuration file is not available"
```

Comment:
Change to "The gammasim-tool configuration file, config.yml, is not available" to be cleater.

Reply:
Done